src/components/bookCard/BookCard.css

```diff
@@ -3,13 +3,17 @@
     width: 300px;
     height: 450px;
     display: flex;
+    flex-direction: column;
     justify-content: center;
     margin-bottom: 15px;
-    color: #fce4db;
+    background-color: #849d9b;
+    padding-left: 10%;
+    padding-right: 10%;
 }
 
 @media (min-width: 750px) {
     .card {
         margin-left: 100px;
```

Comment:
You don't want to insert gaps between flex items like this but this PR is for login so just keep it in mind when you start styling books page